pkg/agent/openflow/client_test.go

```diff
@@ -289,6 +289,199 @@ func Test_client_InstallTraceflowFlows(t *testing.T) {
 	}
 }
 
+func Test_client_SendTraceflowPacket(t *testing.T) {
+	type args struct {
+		dataplaneTag uint8
+		srcMAC       string
+		dstMAC       string
+		srcIP        string
+		dstIP        string
+		IPProtocol   uint8
+		ttl          uint8
+		IPFlags      uint16
+		TCPSrcPort   uint16
+		TCPDstPort   uint16
+		TCPFlags     uint8
+		UDPSrcPort   uint16
+		UDPDstPort   uint16
+		ICMPType     uint8
+		ICMPCode     uint8
+		ICMPID       uint16
+		ICMPSequence uint16
+		inPort       uint32
+		outPort      int32
+	}
+	tests := []struct {
+		name        string
+		args        args
+		wantErr     bool
+		prepareFunc func(*gomock.Controller, bool) *client
+	}{
+		{
+			name:        "err noSrcMAC",
+			args:        args{},
+			wantErr:     true,
+			prepareFunc: prepareSendTraceflowPacket,
```

Comment:
prepareSendTraceflowPacket seems not variable in the test, why don't just use it in the test code but repeat it for each case.

Reply:
OK, removed prepareFunc and call prepareSendTraceflowPacket in for-loop.